content/scout/exceptions.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Manage vulnerability exceptions
```

Comment:
To begin with, we don't have in-product help to deal with incorrect VEX statements (eg. wrong CVE ID, malformed statement). We could, however, suggest that users make use of the CLI VEX capabilities to ensure their statement is correct before adding to a registry. What do you think? Maybe a final section on "Debugging VEX statements" or "Fixing problems with VEX statements" or "Testing VEX statements locally" that points the user towards this capability might be useful?

Reply:
Probably a good idea. I am not very well-versed with the `vexctl` cli to be able to provide any useful tips on how to e.g. debug a malformed vex statement. @chrispatrick do you have any suggestions for what we could put in a section like this?

Reply:
@dvdksn Sorry, I wasn't very clear with my suggestion there. I was referring to the `docker scout cves` command and the `--vex-location` flag you documented previously. If a user had a local image, and a VEX file in a directory, then they ran this and it *didn't* do what they expected, that would give them some feedback there was a problem with their file.

Reply:
@dansmithy I added a section at the bottom of the page... let me know what you think